tests/puppeteer/campaigns/functional/BO/13_shopParameters/03_productSettings/02_productPage/05_separatorOfAttributeAnchor.js

```diff
@@ -0,0 +1,88 @@
+require('module-alias/register');
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_productSettings_separatorOfAttributeAnchorOnProductLinks';
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ProductSettingsPage = require('@pages/BO/shopParameters/productSettings');
+const ProductPage = require('@pages/FO/product');
+const HomePage = require('@pages/FO/home');
+const SearchResultsPage = require('@pages/FO/searchResults');
+// Importing data
+const {Products} = require('@data/demo/products');
+
+let browser;
+let page;
+const attributesWithSeparator2 = '1,size,s/8,color,white';
+const attributesWithSeparator1 = '1-size-s/8-color-white';
```

Comment:
```suggestion
const attributesWithSeparator = ['1', 'size', 's/8', 'color', 'white'];
```
Just a suggestion. See what I suggested below for your `forEach`.